spec/requests/publish_intent_requests_spec.rb

```diff
@@ -27,7 +30,7 @@
     suppresses_draft_content_store_502s
     accepts_root_path
 
-    def put_content_item(body: content_item.to_json)
+    def do_request(body: request_body)
```

Comment:
Is this method necessary, since Actions#do_request uses `request_path` - can we not just define that value in this context?